src/tictactoe_clojure/board.clj

```diff
@@ -1,3 +1,21 @@
 (ns tictactoe-clojure.board)
 
-(def new-board [0 1 2 3 4 5 6 7 8])
\ No newline at end of file
+(def new-board [0 1 2 3 4 5 6 7 8])
+
+(defn valid-slots [board]
+	(filter integer? board))
+
+(defn valid-move?
+	[board position]
+	(some #(= position %) (valid-slots board)))
+
+(defn move [board position symbol]
+	(if (valid-move? board position)
+		(assoc new-board position symbol)
+		"Invalid Move"))
+
+(defn whose-turn
+	[current-turn]
```

Comment:
Similar question here about the board's responsibility and whether or not it should know whose turn it is.